### PR TITLE
[Wiki] Add Release Testing Guide

### LINKED ISSUE #5777: [meta] add default Testing Plan for all module tests
Per Loris mtg Nov26: 
Make a meta test plan, committed to the repo, stating the steps everyone should do when executing manual test plans 
* Check the Help text on every in each page- does it render ok, does the content need updating ? (@ridz1208 your wording suggestion here)
* Test on `both` Chrome and Firefox.  
* Check any other docs for required updates:  the module spec readme.  the test plan itself. 
* Go to the module code, look at all the permissions mentioned in the code, and test what happens with all of them.

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes

Adds an overview of the release testing procedure.

#### Links to related tickets (GitHub, Redmine, ...)

* Resolves #5777 
* Related https://github.com/aces/Loris/issues/4172

### CODE REVIEW

**Comment by johnsaigle** [comment]
To be fleshed out after tomorrow's post-mortem.

Will contain basic LORIS testing information as well as considerations from #5777.

File: docs/wiki/99 - Developers/Release Testing Guide.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Release Testing Guide
+
+
```
Comment (christinerogers) [inline docs/wiki/99 - Developers/Release Testing Guide.md]:
```suggestion
* Browser: test on both Chrome and Firefox
* Help text: Does it render ok and does the content need updating? 

* Install-related? Test for both Ubuntu and CentOS - also check this version number against the Readme/s (which may need updating)
```

Comment (johnsaigle) [reply docs/wiki/99 - Developers/Release Testing Guide.md]: I like the addition but I actually think that the install should have more space devoted to it. It should mention explicitly that an Ubuntu and CentOS install should be tested as well as a development install (i.e. to make sure we have all the dependencies up to date).

Comment (christinerogers) [reply docs/wiki/99 - Developers/Release Testing Guide.md]: yes, definitely 👍

Comment (johnsaigle) [reply docs/wiki/99 - Developers/Release Testing Guide.md]: I'll apply your suggestion and we can iterate further.

**Comment by johnsaigle** [comment]
Unfortunately I wasn't present for the post-mortem meeting and there aren't any notes from that meeting. I don't know if any big things were requested so I'll write this out the best I know how.

**Pull request closed without merging**